acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -197,9 +205,15 @@ public void startNode(final BesuNode node) {
             .besuPluginContext(new BesuPluginContextImpl())
             .build();
 
+    // in testing, we don't mind logging on success, but we want max info on failures
+    // log level is normally set in BesuCommand, which is bypassed here. So let's just set it.
+    // This still leaves out the vertx system properties set in Besu.main().
+    Configurator.setAllLevels("", Level.DEBUG);
```

Comment:
Does this also change the logging levels of the third party dependencies e.g Vertx?

Reply:
This overrides all log4j configuration, including any config files the user has setup.  If we want to enable debug logging we should just change the log4j2.xml file used by acceptance tests.  Then users running locally could still use system properties to use a custom config and use settings they prefer.

Reply:
This is just what BesuCommand.configureLogging() does too. The purpose is to enable DEBUG logging, and since the ThreadBesuNodeRunner bypasses BesuCommand, I couldn't find other simple way to do this.

Anyway, my understanding is that ThreadBesuNodeRunner is rather on its way out, and in sidechains-besu we indeed no longer use it. So I will just remove this change.

Reply:
BesuCommand.configureLogging does that *only* when the user has specifically set a log level on the command line (ie the user has requested the change).  It was deliberately changed to not set a default level so it doesn't override any external log4j config files.

ThreadBesuNodeRunner makes it much easier to test changes to production code - without it you need to run `gradle installDist` after every change. With it you can just rerun the test in IntelliJ. I haven't heard of plans to remove it and thought it was in use by a number of people to make development faster.

Reply:
FWIW, `gradle acceptanceTest` seems to imply `installDist` anyway.

Reply:
Yes, because running in gradle will use the process runner.  The threaded runner is designed to make it easy to run in IntelliJ where `installDist` is not automatically run.

Reply:
`ThreadBesuNodeRunner` also allows easy IDE step through Besu source during execution of the AT.

Plans to delete the thread runner from Besu are also news to me, is this from a RocketChat thread I missed?

Reply:
What I meant is that if IntelliJ is set to use its gradle runner integration (which I think is the default), then both gradle and IntelliJ work as expected, no matter if using the threaded runner or the process runner. Meaning that installDist should not be needed in any case, as far as I can see.
Re: plans to delete: I didn't say "delete", but maybe I still worded that too strongly? My understanding from when we talked was that the threaded runner might change the behavior or the node, and so it's better to use the process runner - meaning that the default advice is to use the process runner.

Reply:
Anyway, at this point this sounds like a red herring. I'm happy to drop this modification.